docs/installation/working-with-typescript.md

```diff
@@ -46,10 +47,19 @@ ClassicEditor.create( editorPlaceholder ).catch( error => {
 } );
 ```
 
-<info-box warning>
-Custom builds produced by online builder do not provide built-in type definitions yet.
-</info-box>
+### Installing CKEditor 5 packages in your TypeScript project
+
+When you use TypeScript you need to import all modules provided by CKEditor 5 using a package entry point instead of a path to a module.
+
+```ts
+// Instead of:
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+// Do:
+import { Plugin } from '@ckeditor/ckeditor5-core';
+```
 
+Thanks to this approach you will avoid problems with missing types during the compile time of your project.
```

Comment:
```suggestion
This approach ensures that TypeScript correctly loads all module augmentation code necessary to make certain types work. The previous method (importing via `src/*`) still works in most cases, but [it may randomly break](https://github.com/ckeditor/ckeditor5/issues/13433).

```